trunk/SUAVE/Input_Output/OpenVSP/vsp_wing.py

```diff
@@ -152,8 +152,18 @@ def read_vsp_wing(wing_id, units_type='SI',write_airfoil_file=True):
     segment_spans    = [None] * (segment_num) 	        # Non-projected.
     segment_dihedral = [None] * (segment_num)
     segment_sweeps_quarter_chord = [None] * (segment_num) 
-    start = 0
-    root_chord = total_chord * units_factor
+
+
+    # Check for wing segment *inside* fuselage, then skip XSec_0 to start at first exposed segment.
+    if np.isclose(total_chord,1):
```

Comment:
This does fix the conversion but I suspect that this if statement condition is always true. I converted a VSP wing component by itself and the `total_chord` was equal to 1, which means the condition was true even without a fuselage. I think the zeroth XSEC for a wing component is not part of the actual physical wing segment definitions (or at least not needed for the conversion). But because I do not know exactly what the zeroth XSEC defines, I cannot say for sure that the if statement condition is always true.

Reply:
This is my suggestion for now: delete lines 145 to 148. And then replace the entire if statement with:
`# Necessary wing segment definitions start at XSec_1 (XSec_0 exists mainly to hold the root airfoil)`
`start = 1`
`xsec_surf_id = vsp.GetXSecSurf(wing_id, 0)`
`x_sec = vsp.GetXSec(xsec_surf_id, 1)`
`chord_parm = vsp.GetXSecParm(x_sec,'Root_Chord')`
`root_chord = vsp.GetParmVal(chord_parm) * units_factor`

Then in the multi-segment wing for loop on lines 177 to 180, delete everything except line 178 (i.e leave `jj = i-1`).

Also, the variable `start` could just be replaced with a value of 1 everywhere in the function.